### DISCUSSION TITLE
Error - Row 4 lag: "ae0" is not a unique value for this field; multiple objects were found

### DISCUSSION THREAD

**Discussion by SatinderOTW** [netbox-community/netbox#8267] [General]
I am trying to do a bulk import of interfaces but keep getting error where it says ae0 is not a unique field. I understand it should be unique per device but it should not be unique in the database as all switches can have ae0. 

This is not a problem for manual entries. 

Netbox version is 3.1.2.

**Comment by jeremystretch** [comment]
Does the device you're adding it to already have an `ae0` interface on it?

**Reply by SatinderOTW** [reply]
No it does not have an ae0 interface on it. I 've tried adding interfaces for different devices but it doesn't work for any of them.

**Reply by jeremystretch** [reply]
It would be helpful if you could share the data you're trying to import.

**Reply by pv2b** [reply]
See https://github.com/netbox-community/netbox/issues/13449

**Comment by candlerb** [comment]
"multiple objects were found": do you have multiple devices with the same name?

If that's not it, then you'll have to drill down in parts.  First, show the exact CSV (header row and data row) you're trying to import, then we can show some SQL queries you can do to look for conflicts.

Is ae0 an interface of type "LAG", or is this an interface which is a member of a LAG?  This should be clear from the CSV.
